src/components/tips/tip-card.tsx

```diff
@@ -29,24 +27,15 @@ const TipCard: React.FC<{tip: Tip; i: number}> = ({tip, i}) => {
   const tipTopic = tip?.tags ? tip.tags[0]?.name : ''
 
   return (
-    <Link
+    <article
```

Comment:
don't wrap the whole item in a link because you get nested a tags which is a no no